docs/supported_platforms_and_configurations.md

```diff
@@ -1,9 +1,24 @@
 # Supported platforms and configurations
 
+The pipeline iterates a fixed list of profiles for every Conan reference, 
+it computes the packageID for each profile and discard duplicates. Then it 
+builds the packages for the remaining profiles and upload them to 
+[JFrog ConanCenter](https://conan.io/center/) once the pull-request is merged.
+
+Because duplicated packageIDs are discarded, the pipeline iterates the 
+profiles always in the same order and the profiles selected to build when 
+there is a duplicate follow some rules:
+
+ * Static linkage (option `shared=False`) is preferred over dynamic linking.
+ * On Windows, `MT/MTd` runtime linkage goes before `MD/MDd` linkage.
+ * Optimized binaries (`build_type=Release`) are preferred over _debug_ ones.
+ * Older compiler versions are considered first.
+ * In Linux, GCC is iterated before Clang.
+
+Currently, given the following supported platforms and configurations we 
+are generating around 130 different binary packages for a C++ library 
+and 90 for a C library. 
```

Comment:
No more accurate numbers? The statistics nerd in me is disappointed 😄

Reply:
I'll come back with accurate numbers, but I can't promise we will keep it updated. Some time ago we added `MTd+Debug` and we didn't update this number, and probably we will forget about it when we add a new compiler version.

Reply:
Oh! I was wrong, @danimtb  took those changes into account. So back to the concrete numbers 😄